tests/population/test_mortality.py

```diff
@@ -83,13 +92,13 @@ def test_mortality_rate(setup_sim_with_pop_and_mortality):
 
 def test_mortality_updates_population_columns(setup_sim_with_pop_and_mortality):
     sim, bp, mortality = setup_sim_with_pop_and_mortality
-    pop0 = sim.get_population()
+    columns_to_update = ["cause_of_death", "exit_time", "years_of_life_lost"]
```

Comment:
Nit: I know this is what it was called before, but I think `updated_columns` is a more intuitive name than `columns_to_update`